### DISCUSSION TITLE
What is the reference font x-height?

### DISCUSSION THREAD

**Discussion by vxncetxn** [Myndex/SAPC-APCA#116] [Q&A] [answered]
There's conflicting information about this in the Myndex docs, is it 0.5 or 0.52?

For example, it is stated as 0.52 here [https://github.com/Myndex/SAPC-APCA/blob/master/documentation/APCA_in_a_Nutshell.md](https://github.com/Myndex/SAPC-APCA/blob/master/documentation/APCA_in_a_Nutshell.md) but 0.5 here [https://www.myndex.com/APCA/](https://www.myndex.com/APCA/).

**Comment by Myndex** [comment, marked as answer]
Hi @vxncetxn thank you for the question.

### The reference x-height is **0.52** vs the font-size.
The same as **Helvetica Neue**, which is the reference font.

**The draft guidelines are now at https://readtech.org/ARC/**

Sorry for the confusion, this is a big project, and we are still in progress as far as developing and improving, so there are bound to be some tiny discrepancies—and 0.5 vs 0.52 is not materially different. There's much more variation than that from one font design to another.

Also, sorry for the delay, just saw your post today. But please feel free to ask any other questions!

**Comment by dj-kmuldoon** [comment]
Any success pulling x-height from the font file meta-data? https://opentype.js.org/font-inspector.html
